planner/core/planbuilder.go

```diff
@@ -79,6 +79,12 @@ type aggHintInfo struct {
 	preferAggToCop bool
 }
 
+// PrepareObject store prepared ast from PrepareExec and other related fields
+type PrepareObject struct {
```

Comment:
1. It's better to put this struct definition together with the prepared plan definition.
2. How about s/PrepareObject/CachedPrepareStmt/`?